lib/Test2/Harness/Renderer/JSONDump.pm

```diff
@@ -0,0 +1,122 @@
+package Test2::Harness::Renderer::JSONDump;
+use strict;
+use warnings;
+
+our $VERSION = '0.000012';
+
+use File::Basename qw/basename dirname/;
```

Comment:
Nope, all core. If non-core was okay it'd be nice to use Path::Tiny or something, but this works well enough.